communicator/ssh/communicator.go

```diff
@@ -343,7 +345,7 @@ func (c *Communicator) Start(cmd *remote.Cmd) error {
 	session.Stdout = cmd.Stdout
 	session.Stderr = cmd.Stderr
 
-	if !c.config.noPty {
+	if !c.config.noPty && c.connInfo.TargetPlatform != "windows" {
```

Comment:
It would be nice to extract these instances of the string `"windows"` (and elsewhere, `"unix"`) into consts if possible.